ci/build-mingw64.sh

```diff
@@ -199,27 +261,33 @@ fi
 CFLAGS+=" -I'$prefix_dir/include'"
 LDFLAGS+=" -L'$prefix_dir/lib'"
 export CFLAGS LDFLAGS
-rm -rf build
+build=mingw_build
+rm -rf $build
 
-meson setup build --cross-file "$prefix_dir/crossfile" \
+meson setup $build --cross-file "$prefix_dir/crossfile" \
     --buildtype debugoptimized \
     -Dlibmpv=true -Dlua=luajit \
     -D{shaderc,spirv-cross,d3d11,libplacebo}=enabled
 
-meson compile -C build
+meson compile -C $build
 
 if [ "$2" = pack ]; then
     mkdir -p artifact
     echo "Copying:"
-    cp -pv build/player/mpv.com build/mpv.exe "$prefix_dir/bin/"*.dll artifact/
-    # ship everything and the kitchen sink
+    cp -pv $build/player/mpv.com $build/mpv.exe artifact/
+    # everything and the kitchen sink
+    cp -pv "$prefix_dir/bin/"*.dll artifact/
     shopt -s nullglob
     for file in /usr/lib/gcc/$TARGET/*-posix/*.dll /usr/$TARGET/lib/*.dll; do
         cp -pv "$file" artifact/
     done
+    # delete unneeded DLLs
+    for file in artifact/lib{gomp,gfortran,objc,quadmath,atomic,harfbuzz-subset,SPIRV-Tools,charset}*.dll; do
```

Comment:
I would prefer listing what we use instead.

Also we shouldn't ship our `vulkan-1.dll`, but IIRC libplacebo does not support dynamically loading it. And on systems without vulkan loader provided by driver we would not have it.

Reply:
> I would prefer listing what we use instead.

A bit annoying to code, not worth it IMO

>  IIRC libplacebo does not support dynamically loading it.

via vk-get-proc-addr libplacebo supports not directly linking libvulkan, but mpv does not support this mode of operation.

Reply:
> A bit annoying to code, not worth it IMO

Yep, I mean simple list. Automating this is indeed not worth it.

> via vk-get-proc-addr libplacebo supports not directly linking libvulkan, 

Right you can pass `get_proc_addr` in `pl_vk_inst_params`, but otherwise it wouldn't try to dlopen by itself. Unlike for GL. 

> but mpv does not support this mode of operation.

Anyway probably improving on mpv is out of the scope of this PR.

Reply:
> Yep, I mean simple list. Automating this is indeed not worth it.

I know what you meant, but now you've convinced me. Will do.

> Anyway probably improving on mpv is out of the scope of this PR.

Exactly :)